frontend/src/components/common/RadioGroup.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+    <div class="row">
+        <b-form-radio-group
+            v-model="selectedValue"
+            :options="props.options"
+            @change="valueChanged"
+        ></b-form-radio-group>
+    </div>
+</template>
```

Comment:
Hi Nick,
I am not sure it is necessary we need to create a wrapper around the library. 
Isn't it easier to just use original <b-form-radio-group> with the doc instruction provided?
I don't see we are styling it differently either.
The doc also can have <b-form-radio-group> displayed in a row just fine.
Or maybe I am missing some points here?

Reply:
The point is to make it easier to style using the carbon template in the future, but if we agree that it is not necessary, I can remove it.

Reply:
Hi Ian, sorry I didn't get this, what means a wrapper around the library? like the line <div class="row"> ?